raymon/tests/profiling/test_sharpness.py

```diff
@@ -0,0 +1,31 @@
+from raymon.profiling.extractors.vision import Sharpness
+
+
+def test_extract(images):
+    assert isinstance(Sharpness().extract(images[0]), float)
+
+
+def test_extract_multiple(images):
+    avg_intensity_list = Sharpness().extract_multiple(images)
+    assert isinstance(avg_intensity_list, list)
+    assert isinstance(avg_intensity_list[0], float)
+
+
+def test_to_jcr():
+    jcr = Sharpness().to_jcr()
+    assert jcr["class"] == "raymon.profiling.extractors.vision.sharpness.Sharpness"
+    assert not jcr["state"]
+
+
+def test_from_jcr():
+    extractor = Sharpness()
+    jcr = extractor.to_jcr()
+    assert isinstance(extractor.from_jcr(jcr), Sharpness)
```

Comment:
Why don't we need to extract 'state' here?

Reply:
in test_intensity and test_sharpness, jcr['state'] is empty. In others, I extract jcr['state'] before calling from_jcr(). However, I did a standardisation for all extractors, I corrected them.